src/App.js

```diff
@@ -1,25 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Intro from './Intro.js';
+
+// possibly add state to search as well
```

Comment:
remove comments about potential implementation outside PR

hint: you can do `git add -p` to enter an interactive UI that allows you to conditionally add things to commit

Reply:
Removed the comment, added the comment to the Trello page